auth/store.go

```diff
@@ -282,6 +285,10 @@ func (as *authStore) AuthDisable() {
 	}
 }
 
+func (as *authStore) AuthStatus() bool {
```

Comment:
This method itself isn't required. `applierV3backend` can call `IsAuthEnabled()` directly.

Reply:
I will work on these tonight.